samples/sampler/.qa-storybook/preview.js

```diff
@@ -15,6 +15,7 @@ const locales = {
 	'th-TH - Thai, with tallglyph characters': 'th-TH',
```

Comment:
Let's move `he-IL` to `am-ET` below and like
'he-IL - Hebrew, RTL, no meridiem': 'he-IL', looks better.